.env.docker-compose

```diff
@@ -8,6 +8,7 @@ CF_AIG_TOKEN=xxxxxxxxxxxxxxxxxxxxxxx
 CF_AIG_NAME=xxxxxxxxxxxxxxxxxxxxxxx
 CF_TEXT_TO_IMAGE_MODEL=@cf/stabilityai/stable-diffusion-xl-base-1.0
 
+<<<<<<< HEAD
```

Comment:
Looks like you left in conflict markers making the .env file impossible to parse.
```suggestion

```